pkg/sql/logictest/testdata/logic_test/mixed_version_upgrade_preserve_ttl

```diff
@@ -18,7 +18,7 @@ upgrade all
 # Verify that the cluster version upgrades have begun. Note that the first
```

Comment:
nit: It may be worth updating this comment with a note similar to your commit message?

IE: `Verify that the cluster version upgrades have begun by asserting we're no longer on the previous version (23.1).`

Reply:
done!